README.adoc

```diff
@@ -82,6 +77,10 @@ compiler currently runs on the following platforms:
 Other operating systems for the processors above have not been tested, but
 the compiler may work under other operating systems with little work.
 
+❗ From OCaml 5.0 onwards, native compilation is available only on 64-bit
+systems.  Native compilation on 32-bit systems is no longer available, nor
+are there plans to bring it back.
+all architectures.
```

Comment:
```suggestion

```

I think this was a copy-paste left over.

Reply:
Oo indeed - or not enough `dd` 🙂